samples/RhythmGame/src/main/cpp/Game.cpp

```diff
@@ -109,26 +124,34 @@ void Game::stop(){
 }
 
 void Game::tap(int64_t eventTimeAsUptime) {
```

Comment:
Is this called when the user taps the screen? Could call onTap().
Or does it make a tap sound?

Reply:
It's both. It's called when a user taps on the screen and it enqueues a tap sound to be played as soon as possible.

Reply:
I'll keep as is because it matches the diagrams in the docs: https://github.com/google/oboe/tree/add-ffmpeg-option/samples/RhythmGame